lib/parity/environment.rb

```diff
@@ -130,7 +130,9 @@ def git_remote
     end
 
     def heroku_app_name
-      git_remote.split(":").last.split(".").first
+      git_remote.
```

Comment:
I think this could be clearer in a couple of ways:

1) The capturing group in the first `split` isn't used
2) `/|:` is the same as `[/:]` 
3) Ruby's regexp literal syntax allows `/` without escaping.

That gives `git_remote.split(%r{[/:]}).last.split(".").first`.

I think that can be made to reveal intent even more:

```
git_remote.split(%r{[/:]}).last.sub(/\.git$/, "")
```

I checked and it works with both HTTPS and Git Heroku URLs.